class EXIT1(Exception):
    
    def __init__(self):
        Exception.__init__(self)
